README.md

```diff
@@ -21,13 +21,13 @@
 ## The Live Coding Music Synth for Everyone.
 
 Sonic Pi is a *new kind of musical instrument*. Instead of strumming
-strings or whacking things with sticks - you write code - **live**.
+strings or whacking things - you write code - **live**.
```

Comment:
Technically, yes, the phrase here might have slightly smoother rhythm without those words, but at the same time, the original does not really feel 'wrong' to me anyway...